src/_pytest/mark/structures.py

```diff
@@ -102,8 +102,10 @@ def extract_from(cls, parameterset, force_tuple=False):
         else:
             return cls(parameterset, marks=[], id=None)
 
-    @classmethod
-    def _for_parametrize(cls, argnames, argvalues, func, config, function_definition):
+    @staticmethod
+    def _parse_parametrize_args(argnames, argvalues, **kwargs):
```

Comment:
```suggestion
    def _parse_parametrize_args(argnames, argvalues, **_):
```

This makes it clearer that those `**kwargs` are ignored. 😁